sstable/table.go

```diff
@@ -225,11 +226,13 @@ const (
 	levelDBFormatVersion  = 0
 	rocksDBFormatVersion2 = 2
 
-	metaRangeKeyName   = "pebble.range_key"
-	metaValueIndexName = "pebble.value_index"
-	metaPropertiesName = "rocksdb.properties"
-	metaRangeDelV1Name = "rocksdb.range_del"
-	metaRangeDelV2Name = "rocksdb.range_del2"
+	metaRangeKeyName        = "pebble.range_key"
+	metaValueIndexName      = "pebble.value_index"
+	metaPropertiesName      = "rocksdb.properties"
+	metaRangeDelV1Name      = "rocksdb.range_del"
+	metaRangeDelV2Name      = "rocksdb.range_del2"
+	metaBlobRefIndexName    = "pebble.blob_ref_index"
+	metaBlobRefLivenessName = "pebble.blob_ref_liveness"
```

Comment:
this guy is unused, right? I think we use `metaBlobRefIndexName` but not `metaBlobRefLivenessName`